scripts/utils/llvm-gcov.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Sorry, I'll add to the documentation, it's in the commit message, if you're using `llvm` you can't use gcov, but using `llvm-cov` requires arguments which `twister` can't pass. Here's the command:

```
ZEPHYR_TOOLCHAIN_VARIANT=llvm ./scripts/twister -p unit_testing --coverage -i --coverage-tool lcov --gcov-tool $(pwd)/scripts/utils/llvm-gcov.sh -T tests/unit/intmath/
```

My next PR will be to automatically handle `--coverage-tool` and `--gcov-tool` in twister depending on `ZEPHYR_TOOLCHAIN_VARIANT`

Reply:
but `twister` can be used on windows, but on windows you can't assume users has bash installed.

So please provide a way that is cross-platform instead of a bash script.